### DISCUSSION TITLE
nominatim import specs

### DISCUSSION THREAD

**Discussion by lauriemann** [komoot/photon#757] [General]
I’m running the 0.4.3 jar against both the GraphHopper-supplied static photon-db es database and separately against an es database created by importing the world from a local nominatim instance.  Running against the GraphHopper database the results match the results from the public api on komoot.  However, the results from the es created from the nominatim instance import (using the same 0.4.3 java jar) are quite different from the public api on komoot.  Can someone provide the correct nominatim import options which are used when creating the weekly GraphHopper ES dump?

**Comment by lonvia** [comment]
The graphhopper extract are created from a Nominatim planet instance. There are bound to be differences. In what way do your results differ? And how did you import and update the Nominatim database?
